spec/requests/congress_messages_spec.rb

```diff
@@ -0,0 +1,185 @@
+      subject
+      expect(response.body).to include "unable to find a congressional district"
+    end
+
+    describe "it filters targets" do
+      before do
+        stub_congress_forms_find_with_one_rep
+      end
+
+      it "to target bioguide_ids" do
+        campaign = FactoryGirl.create(:congress_message_campaign, :targeting_bioguide_ids)
+        action_page.update_attribute(:congress_message_campaign, campaign)
+        subject
+        expect(response.body).to include("C000880")
+        expect(response.body).not_to include("A000360")
+      end
+
+      it "to target a single chamber" do
+        members.last.update_attributes(chamber: "house", district: 10)
+        campaign = FactoryGirl.create(:congress_message_campaign, :targeting_senate)
+        action_page.update_attribute(:congress_message_campaign, campaign)
+        subject
+        expect(response.body).to include("C000880")
+        expect(response.body).not_to include("A000360")
+      end
+    end
+  end
+
+  describe "#create" do
+    let(:message_attributes) do
+      {
+        common_attributes: {
+          "$NAME_FIRST" => "Joyce",
+          "$NAME_LAST" => "Summers",
+          "$ADDRESS_STREET" => "1630 Ravello Drive",
+          "$ADDRESS_CITY" => "Sunnydale",
+          "$ADDRESS_ZIP5" => "94109",
+          "$EMAIL" => "jsummers@altavista.com",
+          "$NAME_PREFIX" => "Mrs.",
+          "$MESSAGE" => "Impeach Mayor Richard Wilkins III"
+        },
+        member_attributes: {
+          "C000880" => {
+            "$SUBJECT" => "Take Action",
+            "$ADDRESS_STATE_POSTAL_ABBREV" => "CA",
+            "$TOPIC" => "JU",
+          },
+          "A000360" => {
+            "$ADDRESS_STATE" => "CA",
+            "$TOPIC" => "Special_Requests"
+          }
+        },
+        bioguide_ids: "C000880 A000360"
+      }
+    end
+
+    subject do
+      campaign_id = action_page.congress_message_campaign_id
+      post("/congress_message_campaigns/#{campaign_id}/congress_messages",
+           params: message_attributes)
+    end
+
+    before do
+      stub_request(:post, /fill-out-form/).
+        and_return(status: 200, body: "{}")
+    end
+
+    it "successfully submits good input" do
+      subject
+      expect(WebMock).to have_requested(:post, /fill-out-form/).
+        with(body: {
+        "bio_id": "C000880",
+        "fields": {
+          "$NAME_FIRST": "Joyce",
+          "$NAME_LAST": "Summers",
+          "$ADDRESS_STREET": "1630 Ravello Drive",
+          "$ADDRESS_CITY": "Sunnydale",
+          "$ADDRESS_ZIP5": "94109",
+          "$EMAIL": "jsummers@altavista.com",
+          "$SUBJECT": "Take Action",
+          "$NAME_PREFIX": "Mrs.",
+          "$ADDRESS_STATE_POSTAL_ABBREV": "CA",
+          "$MESSAGE": "Impeach Mayor Richard Wilkins III",
+          "$TOPIC": "JU"
+        },
+        campaign_tag: "a campaign tag"
+      })
+    end
+
+    it "returns an error when validation fails" do
```

Comment:
is there a way we can check for this error message rather than just checking that it didn't `post`?